views.py

```diff
@@ -82,6 +82,8 @@ class JournalReviewFormElementViewSet(TransporterViewSet):
     queryset = ReviewFormElement.objects.all()
     serializer_class = serializers.JournalReviewFormElementSerializer
 
+    def get_queryset(self):
+        return self.queryset.filter(reviewform__pk=self.kwargs["parent_lookup_review_form__id"])
```

Comment:
It is not clear to me why this view set in need of get_queryset when it is not the pattern for the others. How is it getting called?